code/game/objects/items/hand_items.dm

```diff
@@ -126,6 +128,32 @@
 	"<span class='hear'>You hear a slap.</span>")
 	return
 
+/obj/item/slapper/attack_obj(obj/O, mob/living/user, params)
+	if(!istype(O, /obj/structure/table))
+		return ..()
+
+	var/obj/structure/table/the_table = O
+	var/is_right_clicking = LAZYACCESS(params2list(params), RIGHT_CLICK)
+
+	if(is_right_clicking && table_smacks_left == initial(table_smacks_left)) // so you can't do 2 weak slaps followed by a big slam
```

Comment:
maybe it should do a weak slam instead if you cant do big slam?

Reply:
Yeah, if you try doing a big slam when you've already done at least one weak slap, it'll just flow down to the else below and perform a weak slap instead